.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
delete this? :P 
Could also add vscode to gitignore config

Reply:
It's pretty wierd, `.vscode` is added to the `.gitignore` but still this appears 🤔

Reply:
Forget about this stupid comment, should be fixed now 👍